lib/install/management/appliance.go

```diff
@@ -495,18 +511,23 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	}
 
 	var info *types.TaskInfo
-	// create appliance VM
-	if d.isVC && d.vchVapp != nil {
-		info, err = tasks.WaitForResult(d.op, func(ctx context.Context) (tasks.Task, error) {
-			return d.vchVapp.CreateChildVM(ctx, *spec, d.session.Host)
-		})
-	} else {
-		// if vapp is not created, fall back to create VM under default resource pool
-		info, err = tasks.WaitForResult(d.op, func(ctx context.Context) (tasks.Task, error) {
-			return d.session.VMFolder.CreateVM(ctx, *spec, d.vchPool, d.session.Host)
-		})
+
+	// Create the VCH inventory folder
+	vchFolder := d.session.VMFolder
+	if d.isVC {
+		d.op.Info("Creating the VCH inventory folder")
+		vchFolder, err = d.session.VMFolder.CreateFolder(d.op, spec.Name)
+		if err != nil {
+			d.op.Debugf("Encountered a failure during creation of the inventory folders : %s", err)
```

Comment:
possibly wordsmith to "VCH Folder creation failed: %s" -- don't need inventory and it's singular